### DISCUSSION TITLE
How to make a badge with three blocks?

### DISCUSSION THREAD

**Discussion by holamgadol** [badges/shields#8632] [Q&A] [answered]
I use badges for Jira tasks and I want to add the third block of task type to a Jira badge. This block should lay between the number and the status of the task.

So it should look like this:

![jira](https://user-images.githubusercontent.com/20232096/202373477-0242bfd6-44cd-4922-b92a-529891731b25.png)

Can I achieve my goal by current custom badges or I need to contribute some new features to shields?

**Comment by PyvesB** [comment, marked as answer]
Our badges only currently support two blocks, as documented in our specification: https://github.com/badges/shields/blob/master/spec/SPECIFICATION.md
